src/test/java/task/EventTest.java

```diff
@@ -0,0 +1,100 @@
+package task;
+
+import exception.DukeIllegalArgumentException;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class EventTest {
+
+    @Test
+    void getStartDateTime_sameDateTime_success() {
+        Event event = new Event("Event", LocalDate.parse("2019-05-10").atStartOfDay());
+        assertEquals(event.getStartDateTime(), LocalDate.parse("2019-05-10").atStartOfDay());
+    }
+
+    @Test
+    void setStartDateTime_sameDateTime_success() throws DukeIllegalArgumentException {
+        Event event = new Event("Event", LocalDate.parse("2019-05-10").atStartOfDay());
+        event.setStartDateTime(LocalDate.parse("2019-06-22").atStartOfDay());
+        assertEquals(event.startDateTime, LocalDate.parse("2019-06-22").atStartOfDay());
+    }
+
+    @Test
+    void getEndDateTime_sameDateTime_success() {
```

Comment:
Great effort in accounting for events that last more than a day or drag on till the next day